package/firmware/intel-microcode/Makefile

```diff
@@ -38,15 +38,15 @@ define Package/intel-microcode
 endef
 
 define Build/Compile
-	IUCODE_TOOL=$(STAGING_DIR)/../host/bin/iucode_tool $(MAKE) -C $(PKG_BUILD_DIR)
+	IUCODE_TOOL=$(STAGING_DIR)/../host/bin/iucode_tool $(MAKE) all -C $(PKG_BUILD_DIR)
 	mkdir $(PKG_BUILD_DIR)/intel-ucode-ipkg
```

Comment:
One last nit! This directory isn't needed anymore. :sweat_smile:

Reply:
How would you recommend changing this line? Removing the `-C $(PKG_BUILD_DIR)` will fail

Reply:
The request here is only for line nr 42.

Reply:
Done